src/librustdoc/lib.rs

```diff
@@ -12,6 +12,7 @@
 #![feature(crate_visibility_modifier)]
 #![feature(never_type)]
 #![feature(once_cell)]
+#![feature(option_result_contains)]
```

Comment:
Just noticed I left this in but removed the use of `Option::contains`. Will remove with the next push.